<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_add_animal</class>
 <widget class="QDialog" name="Dialog_add_animal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add animals</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add a new animal</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>269</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Tiger</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Swallow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rabit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mouse</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Leopard</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cat</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Eagle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Crocodile</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dog</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>400</y>
     <width>121</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Comfirm</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>221</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Set the animal's name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Set the animal's age</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Set the animal's weight</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Set the animal's hobbies</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>271</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit_name"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_age"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_weight"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_hobbies"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
